mne/viz/utils.py

```diff
@@ -2025,8 +2025,7 @@ def _setup_butterfly(params):
         types = np.array(params['types'])[params['orig_inds']]
         if params['group_by'] in ['type', 'original']:
             inds = params['inds']
-            eeg = 'seeg' if 'seeg' in types else 'eeg'
-            labels = [t for t in ['grad', 'mag', eeg, 'eog', 'ecg']
+            labels = [t for t in ['grad', 'mag', 'eeg', 'seeg', 'eog', 'ecg']
```

Comment:
you should be using _DATA_CH_TYPES_SPLIT from:

https://github.com/mne-tools/mne-python/blob/master/mne/io/pick.py#L729